.github/workflows/linux.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        crystal_bootstrap_version: [1.2.2]
+        crystal_bootstrap_version: [1.2.2, 1.3.0, 1.3.1]
```

Comment:
Is it necessary to test against both 1.3.0 and 1.3.1?

Reply:
Testing against `1.3.1` should suffice.

Reply:
Ha, that's a good point. #11716 needs improvement then ;)

Reply:
[`44dee47` (#11716)](https://github.com/crystal-lang/crystal/pull/11716/commits/44dee477147c224f962b80cf09010ea44866250e)